core/src/test/resources/custom-fields.adoc

```diff
@@ -169,14 +170,22 @@ query { Person(name:"Jane") {name, label} }
 .Cypher
 [source,cypher]
 ----
-UNWIND apoc.cypher.runFirstColumnSingle('WITH $name AS name MATCH (p:Person { name: $name }) RETURN p',  {
-	name: $personName
-}) AS person
+CALL {
+	WITH $personName AS name
+	MATCH (p:Person { name: name }) RETURN p AS person LIMIT 1
+}
+CALL {
```

Comment:
why did we have the 2 subqueries here if the 2nd outer just is a pass-through, can we detect that and remove/skip it if it has no actual operations?